ui/apps/platform/src/Containers/Clusters/ClustersPage.js

```diff
@@ -22,7 +25,8 @@ const ClustersPage = ({
         params: { clusterId: selectedClusterId },
     },
 }) => {
-    const { hasReadWriteAccess } = usePermissions();
+    const { hasReadAccess, hasReadWriteAccess } = usePermissions();
+    const hasReadAccessForDelegatedScanning = hasReadAccess('Administration');
```

Comment:
I know these are two moving pieces, but in a follow up this might be a good candidate for `isRouteEnabled('clusters/delegated-image-scanning')` since it is linking to the delegated scanning page.

https://github.com/stackrox/stackrox/pull/7524/files#diff-f1d56abf63164636d057d9393c5d4cdb2da3e5a01cc7589056d2da444df010e1R152